src/lib/coda_graphql/coda_graphql.ml

```diff
@@ -1189,6 +1189,17 @@ module Types = struct
             ~args:Arg.[]
             ~resolve:(fun {ctx= coda; _} {With_hash.data; _} ->
               AccountObj.get_best_ledger_account_pk coda data.creator )
+        ; field "winner" ~typ:(non_null public_key)
+            ~doc:"Public key of account that won the slot (Delegator/staker)"
+            ~deprecated:(Deprecated (Some "use creatorAccount field instead"))
```

Comment:
Can you change this to winnerAccount instead? (And make sure you don’t have deprecated)

Reply:
There is a winnerAccount below, so just removing this one